Buildings/Controls/OBC/CDL/Psychrometrics/SpecificEnthalpy_TDryBulPhi.mo

```diff
@@ -0,0 +1,118 @@
+within Buildings.Controls.OBC.CDL.Psychrometrics;
+block SpecificEnthalpy_TDryBulPhi
+  "Block to compute the specific enthalpy based on relative humidity"
+
+  parameter Real pAtm(
+    final quantity="Pressure",
+    final unit="Pa") = 101325 "Atmospheric pressure";
+
+  Interfaces.RealInput TDryBul(
+    final quantity="ThermodynamicTemperature",
+    final unit="K",
+    final min=100) "Dry bulb temperature"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Interfaces.RealInput phi(final min=0, final max=1)
+    "Relative air humidity"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
+        iconTransformation(extent={{-140,-80},{-100,-40}})));
+
+  Interfaces.RealOutput h(
+    final quantity="SpecificEnergy",
+    final unit="J/kg") "Specific enthalpy"
+    annotation (Placement(transformation(extent={{100,-20},{140,20}})));
+
+protected
+  Modelica.SIunits.Conversions.NonSIunits.Temperature_degC TDryBul_degC
+    "Dry bulb temperature in degree Celsius";
+  Modelica.SIunits.Pressure p_w(displayUnit="Pa") "Water vapor pressure";
+  Modelica.SIunits.MassFraction XiDryBul(nominal=0.01)
+    "Water vapor mass fraction at dry bulb state";
+
+equation
+  TDryBul_degC = TDryBul - 273.15;
+  p_w = phi * Buildings.Utilities.Psychrometrics.Functions.saturationPressure(TDryBul);
+  XiDryBul = 0.6219647130774989*p_w/(pAtm-p_w);
+  h = 1006*TDryBul_degC + XiDryBul*(2501014.5+1860*TDryBul_degC);
+
+    annotation (
+    defaultComponentName="ent",
+    Documentation(info="<html>
+<p>
+This block computes the specific enthalpy for a given dry bulb temperature, relative air humidity
+and atmospheric pressure.
+The specific enthalpy is zero if the temperature is <i>0</i>&deg;C and if there
+is no moisture.
+</p>
+<p>
+The correlation used in this model is from the 2005 ASHRAE Handbook Fundamentals, p. 6.9.
```

Comment:
Could we update the reference to `2009 ASHRAE Handbook Fundamentals, p. 1.9, equation 32.`?